### ISSUE TITLE
Multiple Servers?

### ISSUE THREAD

**Issue by elstur** [Beelzebu/Coins#12]
I have this set up on 5 servers under a bungee server atm.

Coins given on server 1 don't immediately reflect on server 2. They add into the DB correctly, but it's like when someone checks how many coins they have or tries to spend them, it doesn't re-check the db to see how much they have. Any idea why?

**Comment by Beelzebu** [comment]
Do you have the plugin installed in bungeecord as well?

**Comment by elstur** [comment]
Nope. It's not clear from the page that this is necessary. I assumed that by linking all servers to the same SQL DB that it would update the info. So to get it to work I need to add it to the bungee as well?

**Comment by Beelzebu** [comment]
There is an installation guide in the wiki https://github.com/Beelzebu/Coins/wiki/Installation

**Comment by elstur** [comment]
There is still a misunderstanding somewhere. Just to make sure:
I have 5 Spigot servers that all have the Coins plugin installed.
I have a Spigot plugin that issues coins on Server 1
On Server 2 I check my coins and it says I have 0.

So in order for this to work, I need to install it on Bungee as well?

The install instructions for Bungee make it sound like I need it to be able to use the API for Bungee plugins, which is not my objective.

Thanks for help/clarification :)
